src/frontend/screens/Invites/InviteReceived.tsx

```diff
@@ -82,6 +88,15 @@ export const InviteReceived = ({
       {
         onSuccess: projectId => {
           setActiveProjectId(projectId);
+
+          if (!activeProjectId) {
+            createProject.mutate(undefined, {
+              onError: err => {
+                Sentry.captureException(err);
+              },
+            });
+          }
```

Comment:
I understand the logic behind checking for an `activeProjectId`, but I think the less fragile check would be to check all the projects, and if there is NOT and a project without a name (aka a solo project) then create a solo project

```ts

const allProjects = useManyProjects()

const hasDefaultProject = allProjects.some(proj=>!proj.name)

...rest
if(!hasDefaultProject) => createDefaultProject
```